config/initializers/hostname.rb

```diff
@@ -0,0 +1,18 @@
+# Compute hostname and port from the hostname in the Settings
+module Hostname
+  def self.fqdn
+    hostname = Settings.hostname.split(':').first
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.